spec/core.md

```diff
@@ -18,187 +18,253 @@ DID, after the prefix, is the case-sensitive [[ref: method-specific identifier]]
-the DID owner might not even be aware. A DID URL can be reused, including by
-hackers. In addition, the same content can be hosted at more than one URL (due
-to redirects, CDNs, load balancers, NAT, IPv4 vs. IPv6, symbolic links,
-/etc/hosts files, intranet DNS overrides, or just redundant publication). This
-can create interpretation challenges for DIDs based on URLs.
-
-To avoid ambiguity, the `did:webs` method adds an additional, [[ref: stable]]
-form of MSI. This is a special [[ref: CESR]]-encoded string datatype that KERI
-calls an [[ref: AID]] ( [[ref: autonomic identifier]]). An AID uniquely and stably
-identifies a DID and its DID doc, regardless of where the DID doc is hosted. The
-DID doc for any `did:webs` DID MUST store the stable form of the MSI in its
-top-level `id` property.
-
-In stable form, `did:webs` MSIs are simply the AID, with no further information.
-Combining such an MSI with the prefix produces a [[ref: stable DID]] that look
-like this:
-
-<pre class="example" title="sample AID → DID transformation">        EmkPreYpZfFk66jpf3uFv7vklXKhzBrAqjsKAn2EDIPM → did:webs:EmkPreYpZfFk66jpf3uFv7vklXKhzBrAqjsKAn2EDIPM
-    </pre>
-
-Stable DIDs don't carry information about how they can be resolved. They are
-thus very similar to URNs, and have limited utility for a party who has never
-resolved them before. However, they can be resolved if the controller of the DID
-passes resolution material directly to the resolver (see [Read
-(Resolve)](#read)). Systems that process `did:webs` MAY also store them as a
-placeholder, to be resolved later. Since the AIDs in stable DIDs lack any
-dependence on web technologies for resolution and discovery, they form the basis
-of a potential interoperability strategy with a DID method that works over other
-protocols.
-
-Across all time, all `did:webs` DID docs that are identified by the same AID (or
-stable DID) MUST be understood to characterize the same DID, and implementations
-of this method MUST require them to be logically consistent with one another. On
-the other hand, DID docs identified by different AIDs MUST be understood to
-characterize different DIDs, even if they are accessed via the same URL (e.g.,
-at different times). This makes accidental or deliberate collisions into errors.
-Implementations MUST treat them as such (e.g., alerting the user that the
-non-stable form of the DID is ambiguous). It also creates the possibility to
-publish the same DID in different places; the non-stable forms of each DID are
-just synonyms. Internal processing of DID docs (which uses only the stable form)
-is deterministic and independent of context that a DID owner may not control.
+:::
+
+::: todo
+
+TODO: Consider merging this and the next section and propose that an entity
+SHOULD only publish one "current" `did:webs`, with defined support for redirects
+Copies of a the `did:webs` data should be just that -- copies.
```

Comment:
This would be at odds with our conclusion that a website is a witness. You can have many witnesses, and there isn't one that's primary over the others. (But things that aren't witnesses would be rejected.)